backend/routes/offline-payment.js

```diff
@@ -8,6 +8,38 @@ const { getConfig } = require('../utils/encryptedConfig')
 const makeOffer = require('./_makeOffer')
 
 module.exports = function (router) {
+  router.post(
+    '/pay/no-charge',
+    authShop,
+    async (req, res, next) => {
+      const { encryptedData } = req.body
+
+      const network = await Network.findOne({
+        where: { networkId: req.shop.networkId }
+      })
+      const networkConfig = getConfig(network.config)
+      const shopConfig = getConfig(req.shop.config)
+      const web3Pk = shopConfig.web3Pk || networkConfig.web3Pk
+
+      if (!web3Pk) {
+        return res.status(400).send({
+          success: false,
+          message: 'Error with payment server'
+        })
+      }
+
+      if (!encryptedData) {
+        return res.json({ success: false, message: 'Missing order data' })
```

Comment:
I'm not sure if we have standardize on format for error returned by the back-end yet.But should that return something else than 200? Like a 400 (Bad Request) given that an expected arg is missing?